scripts/coverage_report_jenkins.py

```diff
@@ -51,6 +118,224 @@ def get_build_numbers(client, job_name):
     return [build['number'] for build in client.get_job_info(job_name)['builds']]
 
 
+def merge_coverage_data(ssh, coverage_dir):
+    """Merge coverage data
+
+    Take all the by appliance by process .resultset.json files from
+    the coverage archive and merge them into one .resultset.json file.
+    Expects the coverage archive to have been extracted to the
+    coverage_dir on the appliance to which the ssh client is connected.
+
+    Args:
+        ssh:  ssh client
+        coverage_dir:  Directory where the coverage archive was extracted.
+
+    Returns:
+        Nothing
```

Comment:
This returns a `6` on L148

Reply:
I'm going to change it to raise an exception.   I'd prefer it just be consistent everywhere.   BTW, our exceptions library doesn't really apply to anything but CFME tests (i.e. the exceptions in there don't make sense for this CLI).

Reply:
okay, glad you brought that up. I would _greatly_ prefer that these raise exceptions instead of returning magic integers.

You can quickly and easily define your own exception types by extending the base Exception class, within this module itself if you'd like.

I don't see a reason to not add exceptions to cfme.exceptions that aren't necessarily CFME application specific, if they're useful within the framework.